tartiflette/engine.py

```diff
@@ -96,79 +96,65 @@ def __init__(
         modules=None,
     ) -> None:
         """
-        Create an Engine instance
+        Create an uncooked Engine instance
         """
-        self._error_coercer = None
-        self._modules = None
         self._parser = TartifletteRequestParser()
         self._schema = None
-
-        if (
-            sdl
-            or schema_name
-            or error_coercer
-            or custom_default_resolver
-            or modules
-        ):
-            logger.warning(
-                """
-                the tartiflette Engine() API evolved started the 0.11 version.
-
-                The engine creation is now asynchronous, to give the ability for the community to
-                create plugins. From now, please use the `create_engine` method to create an instance
-                of `Engine()`.
-
-                ```python
-                from tartiflette import create_engine
-
-                engine = await create_engine(
-                    sdl,
-                    schema_name = "default",
-                    error_coercer = None,
-                    custom_default_resolver = None,
-                    modules = None,
-                )
-                ```
-
-                More details on the website: https://tartiflette.io/docs/api/engine#create_engine-prepares-and-cooks-your-engine
-                """
-            )
+        self._schema_name = schema_name
+        self._error_coercer = error_coercer
+        self._custom_default_resolver = custom_default_resolver
+        self._modules = modules
+        self._sdl = sdl
 
     async def cook(
         self,
-        sdl: Union[str, List[str]],
+        sdl: Union[str, List[str]] = None,
         error_coercer: Callable[[Exception], dict] = None,
         custom_default_resolver: Optional[Callable] = None,
         modules: Optional[Union[str, List[str]]] = None,
-        schema_name: str = "default",
```

Comment:
Just a sanity check: is it OK to pass `None` to `SchemaRegistry.register_sdl()` now?

Reply:
No it still not, logic for validation is done in the method, and default should not override a Engine given value. 
schema_name is computed through `schema_name or self._schema_name or 'default'`